lnrpc/rpc.proto

```diff
@@ -167,6 +181,12 @@ message UnlockWalletRequest {
     the first initialization of the wallet.
     */
     int32 recovery_window = 2;
+    
+    /**
+    stateless_init is an optional argument instructing the daemon NOT to
+    create any *.macaroon files.
+    */
+    bool stateless_init = 3;
```

Comment:
is the expectation that the caller has stored the macaroons sent during `Init`? Maybe we could just resend them on unlock as well?

Reply:
It was the expectation that the caller would store the macaroon during `Init`. But you are right, it makes much more sense to return it during `Unlock` too.
That way the client could only keep it in memory and would never need to store it to disk, making it even safer.

Reply:
Turns out it's not a good idea after all (see general discussion in PR). I removed the macaroon from the `unlock` response.